h/tasks/indexer.py

```diff
@@ -47,13 +47,15 @@ def sync_annotations(limit):
 
 
 @celery.task(acks_late=False)
-def report_sync_annotations_queue_length():
-    search_index = celery.request.find_service(name="search_index")
-
-    count = search_index._queue.count(
-        ["storage.create_annotation", "storage.update_annotation"]
-    )
-
-    newrelic.agent.record_custom_metric(
-        "Custom/SyncAnnotations/Queue/API/Length", count
+def report_job_queue_metrics():
+    queue = celery.request.find_service(name="search_index")._queue
+
+    newrelic.agent.record_custom_metrics(
+        [
+            (
+                "Custom/SyncAnnotations/Queue/API/Length",
+                queue.count(["storage.create_annotation", "storage.update_annotation"]),
+            ),
+            ("Custom/SyncAnnotations/Queue/Expired/Length", queue.count(expired=True)),
```

Comment:
I ended up with something _really_ similar to this in ViaHTML with `Queue/<name>/<metric>` and it's quite nice because NR makes it easy to slap them all on the same graph.

I think I used "Size", but otherwise identical idea.